src/workerd/api/http.c++

```diff
@@ -1210,6 +1210,14 @@ jsg::Ref<AbortSignal> Request::getThisSignal(jsg::Lock& js) {
   return newSignal;
 }
 
+kj::Maybe<jsg::Ref<AbortSignal>> Request::getSignalForSubrequest(jsg::Lock& js) {
```

Comment:
@jasnell You reviewed the original PR I think, can you validate this PR as well?